rules-reviewed/azure/eap/eap-to-azure-appservice-datasource-driver.windup.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<ruleset id="eap-to-azure-appservice-datasource-driver"
+         xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            External resources such as data sources are injected via Java Naming and Directory Interface (JNDI). Some such resources may require migration or reconfiguration.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="eap" versionRange="[7,8)" />
+        <targetTechnology id="azure-appservice"/>
+        <targetTechnology id="azure-aks"/>
+        <tag>password</tag>
+        <tag>security</tag>
+    </metadata>
+    <rules>
+        <rule id="eap-to-azure-appservice-datasource-driver-01000">
+            <when>
+                <xmlfile matches="//driver" />

```

Comment:
I think this is too broad a filter. Searching for a `<driver>` element will find non-database related xml elements too, for instance libvirt disk config xml has a driver element. Maybe we can add a pattern for the acceptable contents of the element ( eg postgresql|jdbc|h2|mongodb etc)?